xiblint/__main__.py

```diff
@@ -47,25 +52,40 @@ def main():
     #
     # Process paths
     #
-    success = True
+    errors = []
     for path in config.include_paths:
         for root, _, files in os.walk(path):
             for file_path in [os.path.join(root, file) for file in files]:
                 checkers = config.checkers(file_path)
-                success = process_file(file_path, checkers) and success
+                errors = errors + process_file(file_path, checkers, reporter)
 
-    sys.exit(0 if success else 1)
+    print_errors(errors, reporter)
 
+    sys.exit(0 if errors.count == 0 else 1)
```

Comment:
Did you mean `len(errors) == 0`? ;)

Reply:
I think I couldn't get this one right and was hoping that you could help me with it in a PR but forgot to comment it 🤦‍♂️ thanks 😄